hbase/Dockerfile

```
@@ -44,7 +44,7 @@ RUN --mount=type=cache,id=maven-hbase-${PRODUCT},uid=${STACKABLE_USER_UID},targe
 ###
 ### HBase
 ###
-curl "https://repo.stackable.tech/repository/packages/hbase/hbase-${PRODUCT}-src.tar.gz" | tar -xzC .
+curl --fail "https://repo.stackable.tech/repository/packages/hbase/hbase-${PRODUCT}-src.tar.gz" | tar -xzC .
```

The stackable-base image contains a `.curlrc` file where sensible defaults like `--fail` are already set:
https://github.com/stackabletech/docker-images/blob/f04f12e86ef288638a7b992ab8d6784a10847a29/stackable-base/stackable/curlrc#L24-L26

```suggestion
curl "https://repo.stackable.tech/repository/packages/hbase/hbase-${PRODUCT}-src.tar.gz" | tar -xzC .
```